src/index.ts

```diff
@@ -185,45 +235,97 @@ export async function decryptWithKey<R>(
  * Receives an exported CryptoKey string and creates a key.
  *
  * @param keyString - The key string to import.
- * @returns A CryptoKey.
+ * @returns An EncryptionKey.
  */
-export async function importKey(keyString: string): Promise<CryptoKey> {
-  const key = await window.crypto.subtle.importKey(
+export async function importKey(
+  keyString: string,
+): Promise<EncryptionKey | CryptoKey> {
```

Comment:
Nit: Maybe worth explaining in the description about these two return types, and when to expect each one

Reply:
Done in https://github.com/MetaMask/browser-passworder/pull/49/commits/131974999a2a2af50f047c16689e46f48f28d085